src/gg.regression.unity.experimental.runespawn/Editor/Scripts/PlayModeController.cs

```diff
@@ -68,6 +72,25 @@ public static void StartPlayMode(TcpClient client)
             EditorApplication.delayCall += () =>
             {
                 Application.runInBackground = true;
+
+                _botSequencePath = botSequencePath;
+                // If a bot sequence path is provided, run the sequence after entering play mode
+                if (!string.IsNullOrEmpty(botSequencePath))
+                {
+                    if (!System.IO.File.Exists(botSequencePath))
```

Comment:
@RG-nAmKcAz may have a more specific recommendation, but we should use some method or utility in our bot sequence code for determining if the botSequencePath exists, because it's actually much more complicated than if it exists on file system. There is a case that since this is only running in editor for now, this should technically always work, but I'd rather be more precise. I'd take a look at the existing bot sequence code to see how we check and use the path.

Reply:
Maybe you could just load it before this with the try catch?

Reply:
`BotSequence.LoadSequenceJsonFromPath`

Reply:
We can save the reference to the parsed bot sequence itself instead of saving the path and trying to load it later when it tries to start

Reply:
Switched to loading the sequence itself and saving that.